src/main/java/com/scalar/db/storage/multistorage/MultiStorage.java

```diff
@@ -141,16 +164,18 @@ public void mutate(List<? extends Mutation> mutations) throws ExecutionException
   private DistributedStorage getStorage(Operation operation) {
     Utility.setTargetToIfNot(operation, Optional.empty(), namespace, tableName);
     String fullTaleName = operation.forFullTableName().get();
-    DistributedStorage storage = storageMap.get(fullTaleName);
-    if (storage == null) {
-      return defaultStorage;
+    DistributedStorage storage = tableStorageMap.get(fullTaleName);
+    if (storage != null) {
+      return storage;
     }
-    return storage;
+    String namespace = operation.forNamespace().get();
+    storage = namespaceStorageMap.get(namespace);
```

Comment:
So, if conflicting mapping like below is given, it prefers namespace mapping, right? Or we shouldn't use both mapping format together?
```
scalar.db.multi_storage.table_mapping=user.ORDER:cassandra,user.CUSTOMER:mysql,coordinator.state:cassandra
scalar.db.multi_storage.namespace_mapping=user:cassandra
```

Anyway, I think it would be great if we note it somewhere.

Reply:
Thank you for reviewing this!

If conflicting mapping between a table mapping and a namespace mapping, it prefers the table mapping because table mappings are more specific than namespace mappings.

Sure, I will add it to the Javadoc. Thanks.

Reply:
Ah, sorry, I meant table mapping is preferred. Thank you!